package.json

```diff
@@ -9,22 +9,24 @@
     "metalsmith-concat": "^2.0.0",
     "metalsmith-markdown": "~0.2.1",
     "metalsmith-permalinks": "~0.4.0",
-    "metalsmith-sass": "~0.6.0",
+    "metalsmith-sass": "~0.7.0",
     "metalsmith-swig-helpers": "^1.0.0",
     "metalsmith-templates": "~0.6.0",
-    "metalsmith-templates": "~0.6.0",
     "metalsmith-metadata": "~0.0.1",
     "swig": "^1.4.2"
   },
   "scripts": {
     "build": "metalsmith build",
-    "test": "metalsmith build && npm run-script styleguide",
-    "postinstall": "bower install && find node_modules/ -name '*.info' -type f -delete",
+    "postbuild": "npm run kss && npm run a11y",
+    "test": "npm run build",
     "start": "grunt",
-    "styleguide": "./node_modules/kss-bootstrap/node_modules/kss/bin/kss-node --config='kss-node.json'",
-    "deploy": "grunt deploy"
+    "postinstall": "bower install && find node_modules/ -name '*.info' -type f -delete",
+    "deploy": "grunt deploy",
+    "kss": "./node_modules/kss-bootstrap/node_modules/kss/bin/kss-node --config='kss-node.json'",
+    "a11y": "a11y build/**/*.html > build/styleguide/a11y.txt"
   },
   "devDependencies": {
+    "a11y": "git://github.com/RobLoach/a11y.git#glob",
```

Comment:
The required change was merged into the project: https://github.com/addyosmani/a11y/pull/37